configure.py

```diff
@@ -727,6 +727,46 @@ def get_env(self):
         return env
 
 
+def _build_env_scripts(cfg):
+
+    if _SYSTEM == "Windows":
+        activate_path = os.path.join(cfg.bin_path, "Activate.ps1")
+        ngl_activate_ps1 = textwrap.dedent(
+            f"""\
+            function global:ngli_deactivate() {{
+                deactivate
+                if (Test-Path -Path Env:NGL_DLL_DIRS) {{
+                    Remove-Item -Path Env:NGL_DLL_DIRS
+                }}
+                Remove-Item -Path function:ngli_deactivate
+            }}
+
+            . {activate_path}
+            $Env:NGL_DLL_DIRS="{cfg.bin_path}"
+            """
+        )
+        with open(op.join(cfg.bin_path, "ngli-activate.ps1"), "w") as fp:
+            fp.write(ngl_activate_ps1)
+
+        return
+
+    activate_path = os.path.join(cfg.bin_path, "activate")
+    ngl_activate = textwrap.dedent(
+        f"""\
+        ngli_deactivate() {{
+            deactivate
+            unset NGL_DLL_DIRS
+            unset -f ngli_deactivate
+        }}
+
+        . {activate_path}
+        export NGL_DLL_DIRS="{cfg.bin_path}"
+        """
```

Comment:
do we really want to set this `NGL_DLL_DIRS` thing on macos and linux?

Reply:
NGL_DLL_DIRS is ignored on platforms other than Windows, so it doesn't hurt to set it unconditionally.